### ISSUE TITLE
output of classfication is same !!!

### ISSUE THREAD

**Issue by cjt222** [eragonruan/text-detection-ctpn#21]
@eragonruan i am sorry to bother you again!!! i have maybe 3500 image,and i set batch size 256 and lr 0.0001,after 5000 iter, the output of classfilation is always same, i do not think it is caused by overfit, because i test image in training dataset,and it can detect nothing. but i test output of classfication is ok at early of training!!! do you know what happened?

**Comment by eragonruan** [comment]
@cjt222 do you mean that the classification loss remain unchanged after 5k iters? Is the loss about 0.69？

**Comment by cjt222** [comment]
i mean not the loss, i mean that when i train some iters, i try to test some image. the probability of predict  @is all the same. @eragonruan

**Comment by eragonruan** [comment]
I haven't met this problem before, maybe you can provide more detail about it, like the running results or something else.

**Comment by cjt222** [comment]
![14321](https://user-images.githubusercontent.com/17508662/32527312-5dd72b20-c468-11e7-9e65-d7f47f9d064c.jpg)
just like this image,  i have 3080 images, and i set lr 0.0001,after 60000 iters, it can detect nothing. i change thresh to ouput bbox,but all bboxes on every image is all same,i do not know why, could youhelp me?thank you so much!!!
